app/models/users.py

```diff
@@ -22,7 +22,8 @@ class UserDB(Base):
     last_name = Column(String)
     roles = relationship("RoleDB", secondary=user_role, back_populates="users", cascade="all, delete")
     documents = relationship("DocumentDB", back_populates="owner")
-
+    signatures = relationship("SignatureDB", back_populates="owner")
```

Comment:
U ovom ne treba biti back_populates "owner" vec "signed" jer to je ono sto ti je zapravo bitno